cocos/audio/CMakeLists.txt

```diff
@@ -105,6 +105,78 @@ elseif(ANDROID)
         audio/android/tinysndfile.cpp
         )
 
+elseif(OHOS)
```

Comment:
Does `OHOS` mean harmonyOS or openharmony? It's really confused to me.
Look at this in Cocos Creator: https://github.com/cocos/cocos-engine/blob/v3.8.2/native/CMakeLists.txt#L114

In Cocos Creator, OHOS macro is for HarmonyOS while OPENHARMONY macro is for openharmony OS.

Reply:
HarmonyOS should be HMOS